src/cow-react/modules/wallet/web3-react/hooks/useWalletMetadata.ts

```diff
@@ -3,6 +3,7 @@ import { getWeb3ReactConnection } from '@cow/modules/wallet/web3-react/connectio
 import { ConnectionType } from '@cow/modules/wallet'
 import { useMemo } from 'react'
 
+const WC_GNOSIS_SAFE_APP_NAME = 'WalletConnect Safe App'
```

Comment:
What is WalletConnect Safe app?  This is when some safe App is used to connect to the CoW Safe App through wallet connect?

Reply:
gnosis-safe.io -> walletconnect -> swap.cow.fi

Reply:
ohh i see, they probably changed the name at some point